app/decorators/super_good/solidus_taxjar/spree/order_updater/fire_recalculated_event.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module SuperGood
+  module SolidusTaxjar
+    module Spree::OrderUpdater
```

Comment:
Could we include a commit body that states that this override is here because of a bug in solidus 2.10 and below?

Reply:
It's not a bug, it's more a missing feature. 🙂 We only started firing the order_recalculated event in Solidus 2.11, so on Solidus < 2.11 we need to fire it ourselves.